deploy/docker-swarm/clickhouse-setup/clickhouse-config.xml

```diff
@@ -2,10 +2,7 @@
 <yandex>
     <logger>
         <level>trace</level>
-        <log>/var/log/clickhouse-server/clickhouse-server.log</log>
-        <errorlog>/var/log/clickhouse-server/clickhouse-server.err.log</errorlog>
-        <size>1000M</size>
-        <count>10</count>
+        <console>1</console>
```

Comment:
@prashant-shahi this will not consume disk space in docker temp files to store logs?

Reply:
Yes it eventually will. Nice catch!

We can replace `trace` with `information`  along with that we can have logging options to make sure older logs are cleaned up. _(with Axolo)_

Reply:
I will need some more information on this. Writing to console is I/O blocking and hence slow compared to writing to file. So, if we are ingesting 100K events/s, will it have impact on performance of ingestion?

Can we just turn on error logs so that it does not print for every write? _(with Axolo)_

Reply:
<https://medium.com/spencerweekly/console-output-overhead-why-is-writing-to-stdout-so-slow-b0cc7c88704c> _(with Axolo)_